lib/chef/knife/inspect.rb

```diff
@@ -13,8 +13,20 @@ class Inspect < Knife
 
       banner 'knife inspect'
 
+      CHECKLISTS.map do |checklist|
+        opt_name = checklist.downcase.to_sym
```

Comment:
First of all, I'm sorry it took me so long to review this!

I'm not happy about what this generates for DataBagItems:

```
--[no-]databagitems          Add or exclude databagitems from inspection
```
I think we need a method to do CamelCase to underscore (I really don't want a dependency on `active_support` for that), so we get `--no-data-bag-items`